### PR TITLE
Backup fixes

### PR DESCRIPTION
**Pull request by ejthurgo**
- Resolves issue: https://github.com/fastly/waflyctl/issues/64
- Resolves issue: https://github.com/fastly/waflyctl/issues/65
- Can now specify backup path.
- A few output clean-ups.
- Fixed Exit code after successful completion.

### CODE REVIEW

File: waflyctl.go [hunk 1]
```diff
@@ -14,18 +14,20 @@ import (
...
+	"path/filepath"
 	"strconv"
 	"strings"
 	"time"
+
+	"github.com/BurntSushi/toml"
+	"github.com/sethvargo/go-fastly/fastly"
+	"gopkg.in/alecthomas/kingpin.v2"
+	"gopkg.in/resty.v1"
 )
```
Comment (fgsch) [inline waflyctl.go]: Is this the result of gofmt?

Comment (ejthurgo) [reply waflyctl.go]: It's goimports, which by default runs on save in VS code.
Personally I much prefer this layout/ordering, but if you disapprove, I can disable it.

Comment (fgsch) [reply waflyctl.go]: I think sticking to gofmt makes it equally good or bad for everyone (and will avoid back and forth depending on the editor and customisations you use), so if you don't mind let's stick to it.

Comment (ejthurgo) [reply waflyctl.go]: This output passes straight through gofmt - so there should be no back & forth or issues.
goimports also removes unused dependancies etc. So I'd certainly recommend it. 
But I can disable it in my IDE if you like.

Comment (fgsch) [reply waflyctl.go]: You are right. The space is making the difference.

File: waflyctl.go [hunk 2]
```diff
@@ -2017,7 +2023,8 @@ var (
 	action           = app.Flag("action", "Action to take on the rules list and rule tags. Overwrites action defined in config file. One of: disabled, block, log.").Enum("disabled", "block", "log")
 	apiEndpoint      = app.Flag("apiendpoint", "Fastly API endpoint to use.").Default("https://api.fastly.com").String()
 	apiKey           = app.Flag("apikey", "API Key to use. Required.").Required().String()
-	backup           = app.Flag("backup", "Store a copy of the WAF configuration in "+homeDir()+"/.waflyctl-<service-id>.rules.").Bool()
+	backup           = app.Flag("backup", "Store a copy of the WAF configuration locally.").Bool()
+	backupPath       = app.Flag("backup-path", "Location for the WAF configuration backup file.").Default(homeDir() + "/waflyctl-backup-<service-id>.toml").String()
```
Comment (ejthurgo) [inline waflyctl.go]: Identified issue with default filename not being tokenised with SID. Do not merge

**Comment by fgsch** [comment]
Looks good. 
Can you also update Documentation/USAGE.md (replace your user with something generic)?

**Pull request merged**